### PR TITLE
v1.39 Mobile changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/mobile-changelog.md
Comment (esethna) [inline source/administration/mobile-changelog.md:16]: Were there no other bug fixes in v1.39? Seems like a really tiny release

Comment (amyblais) [reply source/administration/mobile-changelog.md:16]: I added a few more that were recently fixed, but yes it's a smaller release.

**Comment by amyblais** [comment]
@enahum @esethna This will need one review / approval before I'm able to merge, thanks :)

**Review by enahum — APPROVED** [review]

**Pull request merged**
